test/Service/AuthorizationServiceTest.php

```diff
@@ -157,15 +160,15 @@ public function testReturnsTrueForIdentityWhenHasPermissionButNoAssertionsExists
 
     public function testUsesAssertionsAsInstances()
     {
-        $role = new FlatRole('admin');
-        $identity = new Identity([$role]);
+        $role      = new FlatRole('admin');
```

Comment:
It is not default behavior of popular IDE.
It is not in PSR, ZF or Symfony code style.